### ISSUE TITLE
Package request: jitsi

### ISSUE THREAD

**Issue by Hoshpak** [voidlinux/void-packages#4507]
An audio/video SIP VoIP phone and instant messenger written in Java (formerly SIP-Communicator)

web site: https://jitsi.org/
git repository: https://github.com/jitsi/jitsi

**Comment by Vaelatern** [comment]
I tried this before. It's a real pain to build at all.

**Comment by Hoshpak** [comment]
I came to the same conclusion, unfortunately. The way jitsi is currently published it is nearly impossible to properly package it. They ship precompiled jni libraries which (as if that wasn't bad enough) are compiled against very old versions of certain libraries. Building and packaging all the dependencies from source would propably be possible but a long and tedious task since it is not documented where to get and how to build them. I have downloaded the Debian package and extracted it's contents into /usr/local locally on my system for now. It seems to work and if it stops working some day, I will have to find another Jabber client to use on the desktop.

I'm not sure if this request should be closed for hopelessness or be kept open just in case the upstream situation improves some day.
